doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -401,7 +401,7 @@ New Table w/o Compute Intensive column below
 
 | Flavor Capabilities | Reference<br>RM Chapter 4 and 5 | Basic | Network Intensive | 
 |----------|-------------|--------------|-------------|
-| CPU allocation ratio | nfvi.com.cfg.001| In Nova.conf include <br>cpu_allocation_ratio= 4.0 | In Nova.conf include <br>cpu_allocation_ratio= 1.0 |
+| CPU allocation ratio | nfvi.com.cfg.001| In flavor create or flavor set <br>--property cpu_allocation_ratio=4.0 | In flavor create or flavor set <br>--property cpu_allocation_ratio=1.0 |
```

Comment:
I believe what you are after is CPU Policy and already covered by `nfvi.com.cfg.003`. This is a compute host configuration param to allow that.

Reply:
@deebhatia The table is for flavor definitions and not the implementation

Reply:
@pgoyal01 `cpu_allocation_ratio` is not a standard flavor specification. Can you explain what is the intent with this property here?

Reply:
@deebhatia the cpu_allocation_ratio option is specified in  nova.conf and in the host aggregates metadata. Since flavors themselves have the cpu allocation as part of their specification (originally CNTT Basic with cpu_allocation_ratio of 4.0 and Network Intensive with 1.0) it is a good practice to capture that meta data in the flavor also.

Reply:
@deebhatia BTW this section has existed in the chapter for quite a while. Scott only made typo corrections.

Reply:
Understood. I will iterate the same here that for custom flags like host aggregate flags, we should either explicitly mention or prefix them `aggregate_instance_extra_specs` or do both.

Reply:
@pgoyal01 Any suggestion for custom specs?

Reply:
@deebhatia If the AggregateInstanceExtraSpecsFilter is enabled then it will match specifications taht are scoped with "aggregate_instance_extra_specs" (as suggested by you).  The extra_specs as described are correct but maybe we can add a "Note" for the case where the "AggregateInstanceExtraSpecsFilter" is enabled. But this change should be made on its own -- new Issue and PR so that we capture why th change. The current PR is for minor corrections. Do you agree?

Reply:
Adding details about `AggregateInstanceExtraSpecsFilter` can be taken as separate PR.
This PR can just mention that `it is a custom extra spec.`

Reply:
@deebhatia I added the phrase as you suggested.

Reply:
thanks.